pkg/generate/app/dockerimagelookup.go

```diff
@@ -48,6 +48,16 @@ func (r DockerClientSearcher) Search(precise bool, terms ...string) (ComponentMa
 			errs = append(errs, fmt.Errorf("unable to find the specified docker image: %s", term))
 			continue
 		}
+		if term == "scratch" {
```

Comment:
We probably need to open an issue to make sure build froms scratch work correctly.